app/src/test/java/org/simple/clinic/drugs/selection/EditMedicinesScreenLogicTest.kt

```diff
@@ -184,7 +183,6 @@ class EditMedicinesScreenLogicTest {
     //given
     val protocolDrug = TestData.protocolDrug(uuid = UUID.fromString("362c6a00-3ed9-4b7a-b22a-9168b736bd35"), name = drugName)
 
-    whenever(prescriptionRepository.savePrescription(patientUuid, protocolDrug, facility)).thenReturn(Completable.complete())
```

Comment:
Is there a reason we removed it?

Reply:
Yes, it's because this specific test does not involve saving the prescription at all and just opens the dosage sheet. This was an unused stub, probably carried over by a copy-paste of another test.